### PR TITLE
Lemolim Race Patch

### PR DESCRIPTION
**Pull request by Deustodo**
## Additions

Added patches for Scenario, Race, body, gizmo and Pawnkind.

## Changes

Added a Tribal Grenadier Pawnkind to use Stickybomb.

## References

Values for the leather I based one used in Alpha Animals of similar insulation. (chameleon yak)

## Testing

Check tests you have performed:
- [x ] Compiles without warnings
- [x ] Game runs without errors
- [x ] (For compatibility patches) ...with and without patched mod loaded
- [x ] Playtested a colony (8 hours)

### CODE REVIEW

File: Patches/Lemolim Race/Lemolim_Race.xml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+							<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="Lemolim"]/comps</xpath>
+							<value>
+								<li>
+									<compClass>CombatExtended.CompPawnGizmo</compClass>
+								</li>
+								<li Class="CombatExtended.CompProperties_Suppressable" />
+							</value>
+						</li>
+						
+			<li Class="PatchOperationReplace">
```
Comment (N7Huntsman) [inline Patches/Lemolim Race/Lemolim_Race.xml:122]: This should be in its own file.

File: Patches/Lemolim Race/Lemonim_Pawnkind.xml [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+				<min>15</min>
+				<max>30</max>
+				</primaryMagazineCount>
+				<forcedSidearm>
+					<sidearmMoney>
+					<min>500</min>
+					<max>700</max>
+					</sidearmMoney>
+				<weaponTags>
+				<li>MedievalMeleeAdvanced</li>
```
Comment (N7Huntsman) [inline Patches/Lemolim Race/Lemonim_Pawnkind.xml:38]: This should probably be a Neolithic tag, given the low tech level of the rest of their gear.

File: Patches/Lemolim Race/Scenarios_Patch.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ScenarioDef[defName="LemolimColony"]/scenario/parts</xpath>
+				<value>
+					<li Class="ScenPart_StartingThing_Defined">
+						<def>StartingThing_Defined</def>
+						<thingDef>Bullet_45ACP_FMJ</thingDef>
+						<count>35</count>
```
Comment (N7Huntsman) [inline Patches/Lemolim Race/Scenarios_Patch.xml:28]: Increase to 70, please.

File: Patches/Lemolim Race/Scenarios_Patch.xml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ScenarioDef[defName="LemolimTribe"]/scenario/parts</xpath>
+				<value>
+					<li Class="ScenPart_StartingThing_Defined">
+						<def>StartingThing_Defined</def>
+						<thingDef>Ammo_Arrow_Steel</thingDef>
```
Comment (N7Huntsman) [inline Patches/Lemolim Race/Scenarios_Patch.xml:16]: If you would, make these stone arrows and give them 50.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Some feedback.

**Comment by Deustodo** [comment]
Made all the changes. Originally I put 35 ammo for the autopistol because of the this part in the Scenario description.
"Seeing this as an opportunity to escape, the three make a beeline to the escape pods, dodging any threats that came their way. **An autopistol was grabbed by one of them from a fallen pirate's grasp before they escaped.** The prisoner ship exploded shortly after."

Either way, I made the change.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Looks good, but the Lemolin_Leather file needs to be an xml file.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**